include/fmt/core.h

```diff
@@ -185,11 +185,20 @@
 #  define FMT_CLASS_API
 #endif
 #ifndef FMT_API
-#  define FMT_API
+#  if defined(_WIN32)
+#    define FMT_API
+#  else
+#    define FMT_API __attribute__((visibility("default")))
```

Comment:
One more thing: since `__attribute__` is a non-standard construct it should be conditionally compiled, e.g.

```c++
#   if FMT_GCC_VERSION || FMT_CLANG_VERSION
#     define FMT_API __attribute__((visibility("default")))
#     define FMT_EXTERN_TEMPLATE_API FMT_API
#     define FMT_INSTANTIATION_DEF_API
#   else
#     define FMT_API
#   endif
```